modules/tensor_mechanics/doc/content/source/utils/RankTwoScalarTools.md

```diff
@@ -0,0 +1,249 @@
+# Rank Two Scalar Tools
+
+
+## Description
+
+This is a utility class that provides a set of functions to compute scalar
+quantities for stress or strain tensors. This class is not directly invoked in
+the input file, but is called by several other classes that are, including:
```

Comment:
I think you meant to list out the classes here.  That's a good idea.  They would include the 4 material models you are adding, as well as the 2 pre-existing AuxKernels.

After that, then introduce the list of scalar quantities that can be computed.